wp-content/themes/k1-theme/src/styles/abstracts/_variables.scss

```diff
@@ -47,6 +47,9 @@ $color-primary: $color-k1-green;
 $color-primary--dark: $color-k1-green--dark;
 $color-primary--darkest: $color-academy-green;
 
+$quotes-bg: #00644B;
```

Comment:
this doesn't need to exist as it's already declared as $color-k1-green--dark and $color-primary--dark